frontend/pages/admin/UserManagementPage/UserManagementPage.tsx

```diff
@@ -0,0 +1,643 @@
+
+    let userUpdatedFlashMessage = `Successfully edited ${formData.name}`;
+
+    if (userData?.email !== formData.email) {
+      userUpdatedFlashMessage += `: A confirmation email was sent from ${config?.sender_address} to ${formData.email}`;
+    }
+
+    return (
+      userData &&
+      usersAPI
+        .update(userData.id, formData)
+        .then(() => {
+          dispatch(renderFlash("success", userUpdatedFlashMessage));
+        })
+        .then(() => refetchUsers())
+        .catch(() => {
+          dispatch(
+            renderFlash(
+              "error",
+              `Could not edit ${userEditing?.name}. Please try again.`
+            )
+          );
+        })
+        .finally(() => {
+          toggleEditUserModal();
+        })
+    );
+  };
+
+  const onDeleteUser = () => {
+    if (userEditing.type === "invite") {
+      invitesAPI
+        .destroy(userEditing.id)
+        .then(() => {
+          dispatch(
+            renderFlash("success", `Successfully deleted ${userEditing?.name}.`)
+          );
+        })
+        .catch(() => {
+          dispatch(
+            renderFlash(
+              "error",
+              `Could not delete ${userEditing?.name}. Please try again.`
+            )
+          );
+        })
+        .finally(() => {
+          toggleDeleteUserModal();
+          refetchInvites();
+        });
+    } else {
+      usersAPI
+        .destroy(userEditing.id)
+        .then(() => {
+          dispatch(
+            renderFlash("success", `Successfully deleted ${userEditing?.name}.`)
+          );
+        })
+        .catch(() => {
+          dispatch(
+            renderFlash(
+              "error",
+              `Could not delete ${userEditing?.name}. Please try again.`
+            )
+          );
+        })
+        .finally(() => {
+          toggleDeleteUserModal();
+          refetchUsers();
+        });
+    }
+  };
+
+  const onResetSessions = () => {
+    const isResettingCurrentUser = currentUser?.id === userEditing.id;
+
+    usersAPI
+      .deleteSessions(userEditing.id)
+      .then(() => {
+        if (!isResettingCurrentUser) {
+          dispatch(renderFlash("success", "Sessions reset"));
+        }
+      })
+      .catch(() => {
+        dispatch(
+          renderFlash(
+            "error",
+            "Could not reset sessions for the selected user. Please try again."
+          )
+        );
+      });
+    toggleResetSessionsUserModal();
+  };
+
+  const resetPassword = (user: IUser) => {
+    return usersAPI
+      .requirePasswordReset(user.id, { require: true })
+      .then(() => {
+        dispatch(
+          renderFlash(
+            "success",
+            "User required to reset password",
+            usersAPI.requirePasswordReset(user.id, { require: false }) // this is an undo action.
+          )
+        );
+        toggleResetPasswordUserModal();
+      });
+  };
+
+  const renderEditUserModal = () => {
+    if (!showEditUserModal) return null;
```

Comment:
I believe you can just return without returning `null`. A blank `return` will return `undefined`.

Reply:
This is the pattern used on the previous UserManagementPage.jsx and on current ManageHostPage.tsx.

I filed an issue to update ManageHostPage.tsx https://github.com/fleetdm/fleet/issues/3672

Reply:
Yeah I've noticed it around the codebase and have been cleaning it up as I go. Not a biggie, just not needed. Thanks for creating the ticket.